public/index.html

```diff
@@ -46,22 +47,22 @@
 		<fieldset>
 			<legend>VMasker(element).maskPattern(pattern);</legend>
 
-			<h4>Phone</h4>
+			<label for="phone">Phone</label>
 			<input type="text" id="phone">
 
-			<h4>Phone with values</h4>
+			<label for="phoneValues">Phone with values</label>
 			<input type="text" id="phoneValues" value="3141232312">
 
-			<h4>Date</h4>
+			<label for="date">Date</label>
 			<input type="text" id="date">
 
-			<h4>Document</h4>
+			<label for="doc">Document</label>
 			<input type="text" id="doc">
 
-			<h4>Car plate</h4>
+			<label for="carPlate">Car plate</label>
 			<input type="text" id="carPlate">
 
-			<h4>Vehicle Identification</h4>
+			<label for="vin">Vehicle identification</label>
 			<input type="text" id="vin">
 		</fieldset>
 
```

Comment:
@regishideki, have you differentiate `tab` from `space` in your local editor ?
Looks like it has been indent with `tabs`.